Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -2454,10 +2452,10 @@ static inline const RenderLayer* accumulateOffsetTowardsAncestor(const RenderLay
         if (foundAncestorFirst) {
             // Found ancestorLayer before the abs. positioned container, so compute offset of both relative
             // to enclosingAncestorForPosition and subtract.
-            RenderLayer* positionedAncestor = parentLayer->enclosingAncestorForPosition(position);
-            LayoutSize thisCoords = layer->offsetFromAncestor(positionedAncestor);
-            LayoutSize ancestorCoords = ancestorLayer->offsetFromAncestor(positionedAncestor);
-            location += (thisCoords - ancestorCoords);
+            auto* positionedAncestor = parentLayer->enclosingAncestorForPosition(position);
+            auto thisCoords = layer->offsetFromAncestor(positionedAncestor);
+            auto ancestorCoords = ancestorLayer->offsetFromAncestor(positionedAncestor);
+            location.move(thisCoords - ancestorCoords);
```

Comment:
Is this really more efficient?

Reply:
No, move and operator+= are equally fast. I just find out looks awkward to have places with `location.moveBy(...)` (with no alternative, since operator += is only defined to add a LayoutSize, not a LayoutPoint) and others with use `location += ...`. WDYT?

Reply:
I see your point, agreed.